lib/menu.rb

```diff
@@ -0,0 +1,11 @@
+class Menu
+  attr_reader :dishes
+  def initialize(dishes)
+    @dishes = dishes
```

Comment:
It's great that the user can supply their own menu - it makes the code very flexible (I hard-coded my menu),